runelite-client/src/main/java/net/runelite/client/plugins/woodcutting/Tree.java

```diff
@@ -58,7 +59,7 @@
 {
 	REGULAR_TREE(null, TREE, TREE_1277, TREE_1278, TREE_1279, TREE_1280),
 	OAK_TREE(Duration.ofMillis(8500), ObjectID.OAK_TREE, OAK_TREE_4540, OAK_10820),
-	WILLOW_TREE(Duration.ofMillis(8500), WILLOW, WILLOW_10833, WILLOW_10831),
+	WILLOW_TREE(Duration.ofMillis(8500), WILLOW, WILLOW_10829, WILLOW_10833, WILLOW_10831),
```

Comment:
Thanks! Maybe you can swap places of `WILLOW_10833` and  `WILLOW_10831` too so that the entries are ordered.

Reply:
Absolutely! Surprised I didn't see that at the time. I'll take care of that and resubmit it after work. Thanks.